src/metatile_behavior.c

```diff
@@ -1400,3 +1405,19 @@ bool8 MetatileBehavior_IsTrainerHillTimer(u8 metatileBehavior)
     else
         return FALSE;
 }
+
+bool32 MetatileBehavior_IsSignpost(u32 mb)
```

Comment:
I would consider naming the argument `metatileBehavior` per above. This may also be an excellent moment to convert the bool8s and u8s to 32 above, but feel free to consider this out of scope.

Reply:
I started to do this, but a fuck load of these functions have their results cast as u8 and that just cascaded outwards, and I wouldn't feel comfortable without a full game playthrough / automated testing.

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/5044/commits/1ac4d195c22070f5026e72727788dac4e18cb53e